Homework_10_Java_Software_Messenger_Development/Messenger/src/Main.java

```diff
@@ -0,0 +1,118 @@
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Main class for testing the messenger functionality.
+ */
+
+public class Main {
+	public static void main(String[] args) {
+		User hanna = new User("Hanna");
+		User bartek = new User("Bartek");
+		User konrad = new User("Konrad");
+
+		hanna.addContact(bartek);
+		hanna.addContact(konrad);
+		bartek.addContact(hanna);
+		konrad.addContact(hanna);
+
+		System.out.println("Hanna's contacts: ");
```

Comment:
It can make the client code cleaner if you add a method `void printContacts()` to User class as each User object "knows" its contacts

Less direct calls and code duplication in `main` method, better maintainability (if any changes needed they can be done in a single place) a win-win situation from all POVs :)